### DISCUSSION TITLE
use api.getDayNote problem

### DISCUSSION THREAD

**Discussion by xsmyqf** [TriliumNext/Trilium#3237] [Ideas] [answered]
Can I use getDayNote with a specified calendar root? I want to have access to some workspace's calendar in root workspace. Is this possible?

**Comment by zadam** [comment, marked as answer]
Hi, this API method will use a "current" calendar which means the "global" calendar if not hoisted or a workspace calendar if you're hoisted into a workspace with a calendar.

I just added a possibility to specify "rootNote" of a calendar you want to use as an argument: https://github.com/zadam/trilium/commit/4e3d48fe7d5aff36cf87f8634346ed6f8bf532c5
